tox.ini

```diff
@@ -7,5 +7,5 @@ deps =
 	.[deep-learning]
 
 commands = python -m spacy download en_core_web_sm
-           pytest -m  "not (integration)" -s -v --durations=0 --disable-warnings --tb=line --cov=wellcomeml ./tests
+           pytest -m '{env:TEST_SUITE:}' -s -v --durations=0 --disable-warnings --tb=line --cov=wellcomeml ./tests
```

Comment:
is there a default value? what happens locally in the absence of this variable?

Reply:
In the absence of the value it just throws an empty string and runs all test, which is probably what we want locally.